### PR TITLE
UCP: Minor fixes in proto v2

### PR DESCRIPTION
**Pull request by brminich**
## What
minor fixes in ucp 

## Why ?
better debugging experience

### CODE REVIEW

File: src/ucp/proto/proto_common.inl [hunk 1]
```diff
@@ -355,8 +356,11 @@ ucp_proto_request_pack_rkey(ucp_request_t *req, ucp_md_map_t md_map,
     /* For contiguous buffer, pack one rkey
      * TODO to support IOV datatype write N [address+length] records,
      */
-    ucs_assert(dt_iter->dt_class == UCP_DATATYPE_CONTIG);
-    ucs_assert(ucs_test_all_flags(dt_iter->type.contig.memh->md_map, md_map));
+    ucs_assertv(dt_iter->dt_class == UCP_DATATYPE_CONTIG, "dt_class %d",
+                dt_iter->dt_class);
```
Comment (rakhmets) [inline src/ucp/proto/proto_common.inl:360]:
```suggestion
    ucs_assertv(dt_iter->dt_class == UCP_DATATYPE_CONTIG, "dt_class %d (%s)",
                dt_iter->dt_class, ucp_datatype_class_names[dt_iter->dt_class]);
```

File: src/ucp/proto/proto_common.inl [hunk 2]
```diff
@@ -160,7 +160,7 @@ ucp_proto_request_set_stage(ucp_request_t *req, uint8_t proto_stage)
 {
     const ucp_proto_t *proto = req->send.proto_config->proto;
 
-    ucs_assert(proto_stage < UCP_PROTO_STAGE_LAST);
+    ucs_assertv(proto_stage < UCP_PROTO_STAGE_LAST, "stage %u", proto_stage);
```
Comment (yosefe) [inline src/ucp/proto/proto_common.inl:163]: stage=%u

File: src/ucp/proto/proto_common.inl [hunk 3]
```diff
@@ -180,7 +180,8 @@ static void ucp_proto_request_set_proto(ucp_request_t *req,
                                         const ucp_proto_config_t *proto_config,
                                         size_t msg_length)
 {
-    ucs_assert(req->flags & UCP_REQUEST_FLAG_PROTO_SEND);
+    ucs_assertv(req->flags & UCP_REQUEST_FLAG_PROTO_SEND, "flags 0x%x",
```
Comment (yosefe) [inline src/ucp/proto/proto_common.inl:183]: flags=0x%x

File: src/ucp/proto/proto_common.inl [hunk 4]
```diff
@@ -355,8 +356,11 @@ ucp_proto_request_pack_rkey(ucp_request_t *req, ucp_md_map_t md_map,
     /* For contiguous buffer, pack one rkey
      * TODO to support IOV datatype write N [address+length] records,
      */
-    ucs_assert(dt_iter->dt_class == UCP_DATATYPE_CONTIG);
-    ucs_assert(ucs_test_all_flags(dt_iter->type.contig.memh->md_map, md_map));
+    ucs_assertv(dt_iter->dt_class == UCP_DATATYPE_CONTIG, "dt_class %d",
```
Comment (yosefe) [inline src/ucp/proto/proto_common.inl:359]: dt_class=%s

File: src/ucp/proto/proto_common.inl [hunk 5]
```diff
@@ -160,7 +160,7 @@ ucp_proto_request_set_stage(ucp_request_t *req, uint8_t proto_stage)
 {
     const ucp_proto_t *proto = req->send.proto_config->proto;
 
-    ucs_assert(proto_stage < UCP_PROTO_STAGE_LAST);
+    ucs_assertv(proto_stage < UCP_PROTO_STAGE_LAST, "stage=%u", proto_stage);
```
Comment (rakhmets) [inline src/ucp/proto/proto_common.inl:163]:
```suggestion
    ucs_assertv(proto_stage < UCP_PROTO_STAGE_LAST, "stage=%"PRIu8, proto_stage);
```

File: src/ucp/proto/proto_common.inl [hunk 6]
```diff
@@ -180,7 +180,8 @@ static void ucp_proto_request_set_proto(ucp_request_t *req,
                                         const ucp_proto_config_t *proto_config,
                                         size_t msg_length)
 {
-    ucs_assert(req->flags & UCP_REQUEST_FLAG_PROTO_SEND);
+    ucs_assertv(req->flags & UCP_REQUEST_FLAG_PROTO_SEND, "flags=0x%x",
```
Comment (rakhmets) [inline src/ucp/proto/proto_common.inl:183]:
```suggestion
    ucs_assertv(req->flags & UCP_REQUEST_FLAG_PROTO_SEND, "flags=0x%"PRIx32,
```

**Comment by brminich** [comment]
@rakhmets, @yosefe, fixed

**Review by rakhmets — APPROVED** [review]

**Review by yosefe — APPROVED** [review]

**Pull request merged**